src/client/scripts/game/chess/engine.js

```diff
@@ -0,0 +1,50 @@
+
+/**
+ * This script runs the chess engine for enginegames
+ */
+
+"use strict";
+
+const engine = (function(){
+
+    /**
+     * Main function of this script.
+     * It gets called as soon as the human player submits a move.
+     * It ends by submitting a move via enginegame.makeEngineMove(move).
+     */
+    async function runEngine() {
```

Comment:
This should probably be fed the gamefile for the parameter, and return a move, instead of calling makeEngineMove itself.